source/extensions/filters/http/ext_authz/config.cc

```diff
@@ -46,15 +46,14 @@ Http::FilterFactoryCb ExtAuthzFilterConfig::createFilterFactoryFromProtoTyped(
     const uint32_t timeout_ms =
         PROTOBUF_GET_MS_OR_DEFAULT(proto_config.grpc_service(), timeout, DefaultTimeout);
     callback = [grpc_service = proto_config.grpc_service(), &context, filter_config, timeout_ms,
-                transport_api_version = proto_config.transport_api_version(),
-                use_alpha = proto_config.hidden_envoy_deprecated_use_alpha()](
```

Comment:
Should we at least throw an EnvoyException with some meaningful message if this is now used?

Reply:
Yes, good idea. I've added this.